src/03-ticket-calculator.js

```diff
@@ -54,7 +55,38 @@ const exampleTicketData = require("../data/tickets");
     calculateTicketPrice(tickets, ticketInfo);
     //> "Entrant type 'kid' cannot be found."
  */
-function calculateTicketPrice(ticketData, ticketInfo) {}
+function calculateTicketPrice(ticketData, ticketInfo) {
+  let finalPrice = 0;
+  if (!['general','membership'].includes(ticketInfo.ticketType)) {
+    return "Ticket type 'incorrect-type' cannot be found."
+    }
+  if (!['child','adult','senior'].includes(ticketInfo.entrantType)) {
+    return "Entrant type 'incorrect-entrant' cannot be found."
+    }
+  
+  for (const extra of ticketInfo.extras) {
+    if (!['movie','education','terrace'].includes(extra)) {
+      return "Extra type 'incorrect-extra' cannot be found."
+      }
+     }
```

Comment:
Your return statements can be more dynamic here. For instance if the `ticketType` was VIP, we would want the error message to be `VIP' cannot be found.`